tests/modules/specs/_create_modulemd.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python
+
```

Comment:
Please can you explain what it does and how to use it?

Reply:
@dmach ^^

Reply:
Let's add following comment to the file:

"""
Generate module metadata for modularity tests.

Input:
* RPMs built under ../modules/$name-$stream-$version/$arch
* profiles defined in $name-$stream-$version/profiles.json

Output:
* ../modules/$name-$stream-$version/$arch/$name-$stream-$version.$arch.yaml
"""

Reply:
Fixed.